target/linux/octeon/base-files/lib/preinit/03_rename_interfaces

```diff
@@ -0,0 +1,15 @@
+rename_interfaces() {
+	. /lib/functions/system.sh
+	. /lib/functions.sh
+
+	case "$(board_name)" in
+		ubnt,edgerouter-4)
+			rename_interface_by_mac lan0 $(macaddr_add $(mtd_get_mac_binary eeprom 0) +1)
```

Comment:
Please use 0x0 instead of 0 for consistency with similar statements elsewhere (location/offset should always be hex).

Reply:
sure, used hex instead.